common/src/main/java/whocraft/tardis_refined/common/block/device/AstralManipulatorBlock.java

```diff
@@ -41,9 +38,31 @@ public InteractionResult use(BlockState blockState, Level level, BlockPos blockP
 
                         astralManipulatorBlockEntity.OnRightClick(itemStack);
```

Comment:
Should really be onRightClick, naming conventions

Reply:
Renamed